### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

The misspellings have been reported at https://github.com/jsoref/rstudio/actions/runs/3865441158#summary-10505711069

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/rstudio/actions/runs/3865441258#summary-10505711226

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 ### Intent
 
-> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 
+> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any GitHub issues that are related.
```
Comment (jsoref) [inline .github/pull_request_template.md:4]: brand

File: src/cpp/core/r_util/RActiveSessionsStorage.cpp [hunk 2]
```diff
@@ -107,7 +107,7 @@ size_t FileActiveSessionsStorage::getSessionCount() const
    return listSessionIds().size();
 }
 
-// Returns a shared pointer to the session storage, or an empty session session storage pointer if it does not exist
+// Returns a shared pointer to the session storage, or an empty session storage pointer if it does not exist
```
Comment (jsoref) [inline src/cpp/core/r_util/RActiveSessionsStorage.cpp:110]: i'm not entirely certain about this

File: src/cpp/core/zlib/inftrees.h [hunk 3]
```diff
@@ -41,7 +41,7 @@ typedef struct {
    examples/enough.c found in the zlib distribtution.  The arguments to that
    program are the number of symbols, the initial root table size, and the
    maximum bit length of a code.  "enough 286 9 15" for literal/length codes
-   returns returns 852, and "enough 30 6 15" for distance codes returns 592.
+   returns 852, and "enough 30 6 15" for distance codes returns 592.
```
Comment (jsoref) [inline src/cpp/core/zlib/inftrees.h:44]: ?

File: src/cpp/ext/fmt/ChangeLog.rst [hunk 4]
```diff
@@ -3588,7 +3588,7 @@
   `#555 <https://github.com/fmtlib/fmt/issues/555>`_).
   Thanks `@alabuzhev (Alex Alabuzhev) <https://github.com/alabuzhev>`_,
   `@virgiliofornazin (Virgilio Alexandre Fornazin)
-  <https://gihtub.com/virgiliofornazin>`_,
+  <https://github.com/virgiliofornazin>`_,
```
Comment (jsoref) [inline src/cpp/ext/fmt/ChangeLog.rst:3591]: notable

File: src/cpp/session/modules/rmarkdown/NotebookAlternateEngines.cpp [hunk 5]
```diff
@@ -795,7 +795,7 @@ Error executeAlternateEngineChunk(const std::string& docId,
    else
    {
       // check to see if this is a known interpreter; if so, we'll
-      // use own own shim to run the engine. if not, we'll just call
+      // use our own shim to run the engine. if not, we'll just call
```
Comment (jsoref) [inline src/cpp/session/modules/rmarkdown/NotebookAlternateEngines.cpp:798]: atypical correction

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_en.properties [hunk 6]
```diff
@@ -216,12 +216,12 @@ vcsFileRevertDesc = Revert changes to the file
 # vcsViewOnGitHub
 vcsViewOnGitHubLabel = View file on GitHub
 vcsViewOnGitHubMenuLabel = _View FILE on GitHub
-vcsViewOnGitHubDesc = View this file on Github
+vcsViewOnGitHubDesc = View this file on GitHub
```
Comment (jsoref) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants_en.properties:219]: I'm not changing localizations...

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java [hunk 7]
```diff
@@ -225,7 +225,7 @@ public boolean shouldComplete(NativeEvent event)
          if (!userPrefs_.tabCompletion().getValue() || userPrefs_.tabKeyMoveFocus().getValue())
             return false;
 
-         // If the user opted in to multi-line tab completion, there is no case where we don't attempt auto-complete.
+         // If the user opted into multi-line tab completion, there is no case where we don't attempt auto-complete.
```
Comment (jsoref) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java:228]: changes for `opt-in` / `into` tend to be controversial...

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine

**Comment by kevinushey** [comment]
Thanks for the PR, and sorry for the delay in reviewing. The PR makes changes to a number of 3rd-party libraries, for which I think the fixes should be committed upstream.

There are also a variety of changes to some string usages in code, e.g. `non-existent` to `nonexistent` which I believe are too aggressive (I think both spellings are acceptable).

For that reason, I'm going to close this PR. Again, I apologize for the delay in reviewing; we'll definitely welcome spellcheck PRs that touch only documentation or code comments.

**Comment by jsoref** [comment]
If you could identify which things are out of scope, I could fairly easily exclude them.

It isn't a problem for me to drop nonexistent or similar.

**Pull request closed without merging**